diff -urpN /tmp/linux-4.9.111/arch/x86/entry/syscalls/syscall_64.tbl linux-4.9.111/arch/x86/entry/syscalls/syscall_64.tbl
--- /tmp/linux-4.9.111/arch/x86/entry/syscalls/syscall_64.tbl	2018-07-03 09:23:18.000000000 +0000
+++ linux-4.9.111/arch/x86/entry/syscalls/syscall_64.tbl	2019-10-11 12:14:22.174949195 +0000
@@ -338,6 +338,7 @@
 329	common	pkey_mprotect		sys_pkey_mprotect
 330	common	pkey_alloc		sys_pkey_alloc
 331	common	pkey_free		sys_pkey_free
+332	common	lin_hello 		sys_lin_hello
 
 #
 # x32-specific system call numbers start at 512 to avoid cache impact
diff -urpN /tmp/linux-4.9.111/kernel/Makefile linux-4.9.111/kernel/Makefile
--- /tmp/linux-4.9.111/kernel/Makefile	2018-07-03 09:23:18.000000000 +0000
+++ linux-4.9.111/kernel/Makefile	2019-10-11 12:13:49.583001210 +0000
@@ -9,7 +9,7 @@ obj-y     = fork.o exec_domain.o panic.o
 	    extable.o params.o \
 	    kthread.o sys_ni.o nsproxy.o \
 	    notifier.o ksysfs.o cred.o reboot.o \
-	    async.o range.o smpboot.o ucount.o
+	    async.o range.o smpboot.o ucount.o lin_hello.o
 
 obj-$(CONFIG_MULTIUSER) += groups.o
 
diff -urpN /tmp/linux-4.9.111/kernel/lin_hello.c linux-4.9.111/kernel/lin_hello.c
--- /tmp/linux-4.9.111/kernel/lin_hello.c	1970-01-01 00:00:00.000000000 +0000
+++ linux-4.9.111/kernel/lin_hello.c	2019-10-11 12:13:32.119034798 +0000
@@ -0,0 +1,8 @@
+#include <linux/syscalls.h>
+#include <linux/kernel.h>
+
+SYSCALL_DEFINE0(lin_hello)
+{
+	printk(KERN_DEBUG "Hello Wolrd!\n");
+	return 0;
+}
